spec/controllers/admin/payments_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe Admin::PaymentsController do
+  describe "GET index" do
+    it_behaves_like 'require_admin' do
+      let(:action) { get :index }
+    end
+
+    it "shows the last 10 records of payment" do
```

Comment:
This description is a bit misleading. You are not verifying that it shows the last ten, you are verifying that it "only shows ten". You do not verify any order.